src/shared/components/form-tags-input/gio-form-tags-input.component.ts

```diff
@@ -53,6 +53,9 @@ export class GioFormTagsInputComponent implements MatFormFieldControl<Tags>, Con
   @Input()
   addOnBlur = true;
 
+  @Input()
+  confirmTagAdded: (tag: string, confirmCb: (confirm: true) => void) => void;
```

Comment:
Maybe for better understanding, we can rename the input and add some doc like:
```suggestion
  /**
   * Function called each time a tag is added, it can be used to hook inside the
   * addition process to, for instance, have custom validation in the components
   * using `gio-form-tags-input`.
   *
   * Parameters are:
   * - `tag`: The value of the tag to add
   * - `validationCb`: The callback function to call when validation is done. If
   * called with `true` it will add the tag, otherwise it will just ignore it
   */
  @Input()
  tagValidationHook: (tag: string, validationCb: (shouldAddTag: boolean) => void) => void;

```

Reply:
Thanks, I wanted to talk about that this morning  SU. 
I've hesitated to change callback by observable, to avoid the function inside function.

but yes thats really better with this wording,  thanks

Reply:
✅